spec/features/api/v1/places_controller_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
```

Comment:
This file belongs in spec/controllers/api/v1/places_controller_spec.rb (it's not a feature)

Reply:
all done.